### PR TITLE
Project modal

### PR DESCRIPTION
**Pull request by mntedev**
we completed the following tasks

1. added mobile project popup modal
2. added desktop  version project popup modal
3. added toggle popup modal windows

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -752,6 +753,99 @@ ul.social-link .icon {
   margin-right: 140px;
 }
 
+.project-modal {
```
Comment (OLIPLICHE) [inline css/style.css:756]: - [ ] Please the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., margins).
- Your project is working fine, I suggest that you change the margin-top and margin-down. Example: "margin: 12px auto"
- Please in adjusting the margin, make sure the popup fits mobile and desktop sizes properly. 
- See the original Design:
g

File: js/modal.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    languages: [
+      'CSS',
+      'HTML',
+      'Bootsrap',
+      'Ruby',
+
+    ],
+    liveLink: '',
+    sourceLink: '',
+    description: " A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
```
Comment (OLIPLICHE) [inline js/modal.js]: - [ ] Please de the description on the "popup" is different from the initial project description. Kindly add more lines in your popup description. ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A838))

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mntedev** [comment]
completed the following tasks
1. update popup window margin
2. update project description, source link and live link

File: css/style.css [hunk 3]
```diff
@@ -752,6 +753,104 @@ ul.social-link .icon {
...
+}
+
+.modal-container {
+  position: fixed;
+  inset: 0 0;
+  background-color: rgba(0, 0, 0, 0.8);
+  width: 100%;
+  height: 100%;
+  display: none;
+}
```
Comment (thecodechaser) [inline css/style.css]: - [ ] You did a good job on the functionalities but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design as much as possible. As requested by the previous reviewer Please increase the top margin and make sure it matches the Figma design. Please add a blur effect in the background.
  - For blur you can use effect `backdrop-filter: blur(5px);`, use px according to your design.

### Your design

image

### Figma design

image

File: css/style.css [hunk 4]
```diff
@@ -752,6 +753,104 @@ ul.social-link .icon {
...
+  max-width: 90%;
+  height: 98vh;
+  padding: 0 10px;
+  margin: 12px auto;
+}
+
+.modal-container {
+  position: fixed;
+  inset: 0 0;
+  background-color: rgba(0, 0, 0, 0.8);
```
Comment (thecodechaser) [inline css/style.css]: - [ ] Please increase the image and project description size so they occupy the empty model height from the bottom and please also make sure your design matches with the Figma design.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @worashf 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by worashf** [comment]
I  made the following change.
 1. corrected the popup margin from the top
image
2. added blur background effect
3. updated project image   and update  project description height to fit the popup window.
image

**Comment by mntedev** [comment]
I made the following changes.
1. updated the margin of the popup window from the top

image

2. updated the margin of the project image and the project description to the correct margin like the figma  template.

image

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @worashf, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**